doc/source/security/data-at-rest-encryption.rst

```diff
@@ -0,0 +1,390 @@
+
+You must add the ``ENCRYPTION`` option to ``ALTER TABLE`` to change the table encryption state. Without the ``ENCRYPTION`` option, an encrypted table remains encrypted or an unencrypted table remains unencrypted.
+
+Encrypting a General Tablespace
+-------------------------------------------
+
+As of :rn:`5.7.20-18`, |Percona Server| supports general tablespace encryption. A general tablespace must be either encrypted; all tables are encrypted, or not encrypted, no encrypted tables. You cannot encrypt only some of the tables in a general tablespace.
+
+.. rubric:: Automatically Encrypting Tablespaces
+
+Add the ``innodb_encrypt_tables`` variable to my.cnf to automatically encrypt general tablespaces. The possible values for the variable are:
+
+.. list-table::
+    :widths: 25 50
+    :header-rows: 1
+
+    * - Value
+      - Description
+    * - OFF
+      - The default value which disables automatic encryption of new tables
+    * - ON
+      - Enables automatic encryption for new tables
+    * - FORCE
+      - New tables are automatically created with encryption. 
+
+        Adding ``ENCRYPTION=NO`` to either a ``CREATE TABLE`` or ``ALTER TABLE`` statement results in a warning.
+
+The `CREATE TABLESPACE <https://dev.mysql.com/doc/refman/5.7/en/create-tablespace.html>`_ statement is extended to allow the ``ENCRYPTION=['
+Y/N']`` option.
+
+.. code-block:: guess
+
+    mysql> CREATE TABLE t1 (id INT) ENCRYPTION='Y';
+
+To encrypt an existing table, add the `ENCRYPTION` option in the ``ALTER TABLE`` statement. 
+
+.. code-block:: MySQL
+
+    mysql> ALTER TABLE t1 ENCRYPTION='Y';
+
+You can also disable encryption for a table, set the
+encryption to `N`.
+
+.. code-block:: MySQL
+
+    mysql> ALTER TABLE t1 ENCRYPTION='N';
+
+.. note::
+
+    The ``ALTER TABLE`` statment modifies the current encryption mode only if
+    the ``ENCRYPTION`` clause is explictily added.
+    
+.. rubric:: System Variables
+
+.. variable:: innodb_encrypt_tables
+
+   :version 5.7.21-21: Implemented
+   :cli: ``--innodb-encrypt-tables``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Text
+   :default: ``OFF``
+
+:Availability: This feature is **Experimental** quality.
+
+Encrypting Binary Logs
+-----------------------
+
+To start binlog encryption, start the server with ``-encrypt-binlog=1``. This state requires ``-master_verify_checksum`` and ``-binlog_checksum`` to be ``ON`` and one of the keyring plugins loaded.
+
+.. note::
+
+    These actions do not encrypt all binlogs in a replication schema. You must enable ``encrypt-binlog`` on each of the replica servers, even if they do not produce binlog files. Enabling encryption on replica servers enable relay log encryption.
+    
+|Availability| The ``rotate_system_key("percona_binlog")`` command is **Experimental** quality.
+
+You can rotate the encryption key used by |Percona Server| by running the
+following statement:
+
+.. code-block:: MySQL
+
+    mysql> SELECT rotate_system_key("percona_binlog");
```

Comment:
This one was never QAed. So I would say that SELECT rotate_system_key("percona_binlog") is of Experimental quality.

Reply:
Moved the Experimental sentence to after the variable definition.